test/serializers/serialization_array_test.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module ActiveModel
+  class Serializer
+    class SerializationArrayTest < ActiveSupport::TestCase
+      class ArraySerializer < ActiveModel::Serializer
+        attributes :total, :items
```

Comment:
@jpawlyn can you put a `type` option here with any name you like and rerun all the tests again. I believe it should pass the tests.
```ruby
type 'array'

attributes :total, :items # you can add any number of fields you like
```

The `type` value is used at the top level of the json if you use the `:json` adapter, but skipped for the `:attributes` adapter.

You can find the details of this option here: https://github.com/rails-api/active_model_serializers/blob/0-10-stable/docs/general/serializers.md#type

Though it says this option is not useful with the `:attributes` adapter, but providing it in the non-resource serializers like yours will help overcome the issue that you are having.

Let me know if you still find any issue after providing this option. Thanks.

Reply:
That's great, thanks a lot for your help with this @wasifhossain, much appreciated. I'll close the PR now.

Reply:
you are most welcome! glad that it worked for you :raised_hands: